package.json

```diff
@@ -48,13 +58,18 @@
     "@storybook/addons": "^3.4.0",
     "@storybook/react": "^3.4.0",
     "@sumup/foundry": "^0.0.8",
+    "babel-cli": "^6.26.0",
     "babel-eslint": "^8.1.2",
     "babel-jest": "^22.1.0",
     "babel-loader": "^7.1.2",
     "babel-plugin-dynamic-import-node": "^1.2.0",
+    "babel-plugin-emotion": "^9.2.6",
+    "babel-plugin-lodash": "^3.3.2",
     "babel-plugin-module-resolver": "^3.0.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
Any reason why you're not using the `env` preset? AFAIK, all the ES version specific presets are deprecated.